jre8/plan.sh

```diff
@@ -6,10 +6,10 @@ pkg_source=http://download.oracle.com/otn-pub/java/jdk/${pkg_version}-b14/jre-${
 pkg_shasum=214ff6b52f5b1bccfc139dca910cea25f6fa19b9b96b4e3c10e699cd3e780dfb
 pkg_filename=jre-${pkg_version}-linux-x64.tar.gz
 pkg_license=('Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX')
-pkg_description=('Oracle Java Development Kit. This package is made available to you to allow you to run your applications as provided in and subject to the terms of the Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX, found at http://www.oracle.com/technetwork/java/javase/terms/license/index.html')
+pkg_description=('Oracle Java Runtime Environment. This package is made available to you to allow you to run your applications as provided in and subject to the terms of the Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX, found at http://www.oracle.com/technetwork/java/javase/terms/license/index.html')
 pkg_upstream_url=http://www.oracle.com/technetwork/java/javase/overview/index.html
 pkg_deps=(core/glibc)
-pkg_build_deps=(core/patchelf core/file)
```

Comment:
Well probably it is not needed, because I was able to have a functional JRE package without that dependency...